pkg/applicationmanager/controllers/applicationbackup.go

```diff
@@ -1141,24 +1141,10 @@ func (a *ApplicationBackupController) backupResources(
 			}
 		}
 	}
-	updatedAllObjects := make([]runtime.Unstructured, 0)
```

Comment:
Dont need this anymore , since `GetResources` and `GetResourceTypes` delete all metadata of objects UID will always be empty